docs/guide/acquiring_data/database.rst

```diff
@@ -179,7 +179,7 @@ In section 3, more advanced formats of the slicing syntax are introduced.
     >>> len(entry.fits_header_entries)
     191
     >>> for fits_header_entry in entry.fits_header_entries[:10]:
-    ...     print('{entry.key}\n\t{entry.value}'.format(entry=fits_header_entry))   # doctest: +NORMALIZE_WHITESPACE
```

Comment:
why all the removals of normalize whitespace in this file?

Reply:
ah nice! I don't think we need the `doctest :` if there is nothing after it though

Reply:
Good point, not sure how i didnt spot that before.